example/src/main/java/io/radar/example/MainActivity.kt

```diff
@@ -190,23 +190,14 @@ class MainActivity : AppCompatActivity() {
             Log.v("example", "Matrix: status = $status; matrix[0][0].duration.text = ${matrix?.routeBetween(0, 0)?.duration?.text}; matrix[0][1].duration.text = ${matrix?.routeBetween(0, 1)?.duration?.text}; matrix[1][0].duration.text = ${matrix?.routeBetween(1, 0)?.duration?.text};  matrix[1][1].duration.text = ${matrix?.routeBetween(1, 1)?.duration?.text}")
         }
 
-        val customEventMetadata = JSONObject()
-        customEventMetadata.put("one", "two")
+        val conversionMetadata = JSONObject()
+        conversionMetadata.put("one", "two")
 
-        Radar.sendEvent(
+        Radar.logConversion(
             "app_open_android",
-            customEventMetadata
-        ) { status, location, events, user ->
-            Log.v("example", "Custom event type = ${events?.first()?.customType}: status = $status; location = $location; events = $events; user = $user")
-        }
-
-        // Send custom event with manual location
```

Comment:
For Tim's ref: it was agreed to remove the ability to send a conversion event with a manual location: https://radarlabs.slack.com/archives/C02MF6PAWBH/p1677793796912529?thread_ts=1677790478.849349&cid=C02MF6PAWBH